### PR TITLE
Added new discount codes and notification.

### PR DESCRIPTION
**Pull request by Vindexus**
localhost_8080_account_gold 1
localhost_8080_

### CODE REVIEW

**Comment by tmathews** [comment]
Screen shots required

**Comment by Vindexus** [comment]
Screenshots added.

File: src/js/shop-codes.js [hunk 1]
```diff
@@ -25,13 +25,13 @@ function transformShopCode (code) {
   }
   code.discountText = (parseInt(code.value) * -1) + '% off';
   if(code.rewardFor == '1month') {
-    code.rewardForText = 'Being Gold'
-  }
-  else if(code.rewardFor == '6months') {
-    code.rewardForText = '6 Months of Gold';
+    code.rewardForText = 'being Gold'
```
Comment (tmathews) [inline src/js/shop-codes.js]: Should start using a map at this point.

**Comment by Vindexus** [comment]
Forgot to add this section to the Gold page.
image

**Pull request merged**